### PR TITLE
Merge overlapping intervals in java

### PR DESCRIPTION
**Pull request by ishitakeshawani**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

### Description

I make a solution to merge overlapping intervals in java using 2d array and ArrayList.

### Checklist

- [✔️] I've read the contribution guidelines.
- [✔️  ] I've checked the issue list before deciding what to submit.
- [✔️  ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

this is a solution of #449

### CODE REVIEW

File: Java/cp/Answer.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package def;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Answer {
```
Comment (iamrajiv) [inline Java/cp/Answer.java]: @Ishita1608 use suitable class name

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Ishita1608 Rename the file `Answer.java` to the suitable file name. Add sample input-output, space and time complexity of the program at end of the file in a multiline comment something like this
```
/*
...
*/
Also, remove whitespaces and indentations

File: Java/cp/Answer.java (hunk 1 above)
Comment (ishitakeshawani) [reply Java/cp/Answer.java]: ok, I got it.

**Comment by ishitakeshawani** [comment]
I have done what you tell. There show some conflicts can you review it?
@iamrajiv

**Review by iamrajiv — APPROVED** [review]
@Ishita1608 LGTM!

**Review by atarax665 — APPROVED** [review]

File: Java/cp/MergeOverlappingArray.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package def;
```
Comment (atarax665) [inline Java/cp/MergeOverlappingArray.java]: Add the description of the problem at the top.

**Review by atarax665 — APPROVED** [review]

File: Java/cp/MergeOverlappingArray.java (hunk 2 above)
Comment (ishitakeshawani) [reply Java/cp/MergeOverlappingArray.java]: Done

**Review by siddharth25pandey — APPROVED** [review]
@Ishita1608 Good to go
Able to merge.
Thanks!

**Review by siddharth25pandey — APPROVED** [review]

**Comment by siddharth25pandey** [comment]
@all-contributors please add @Ishita1608  for this code

**Pull request merged**